### PR TITLE
Add support for multiple brands

### PR DESCRIPTION
**Pull request by SerhiiBovtriuk**

### CODE REVIEW

File: TDConnectIosSdk/IdProvider.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//  TDConnectIosSdk
+//
+//  Created by Serhii Bovtriuk on 29/04/2019.
+//  Copyright © 2019 aerogear. All rights reserved.
+//
+
+import Foundation
+
+public enum IdProvider {
+    case connectId, telenorId
```
Comment (metatelenor) [inline TDConnectIosSdk/IdProvider.swift]: would be nice with a newline here

File: TDConnectIosSdk/IdProvider.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    func getUrl(useStaging: Bool) -> String {
+        switch self {
+            case .connectId:
+                return useStaging ? "https://connect.staging.telenordigital.com/oauth" : "https://connect.telenordigital.com/oauth";
+            case .telenorId:
+                return useStaging ? "https://staging.telenorid.com/oauth" : "example.com"; // TODO: Production is not set yet
```
Comment (metatelenor) [inline TDConnectIosSdk/IdProvider.swift]: Are you going to release this or is this just to prepare for a later release where we have to change this anyway? I don't really like having links to "example.com" in any production code

Comment (metatelenor) [reply TDConnectIosSdk/IdProvider.swift]: You talked about this during the standup. It looks fine if you do not merge this with master until we have finalized the urls and added them

Comment (SerhiiBovtriuk) [reply TDConnectIosSdk/IdProvider.swift]: @metatelenor this will stay here in frozen mode for a while until we are sure about exact URLs of Telenor ID

**Pull request merged**